### ISSUE TITLE
Sshuttle in combination with SSH remote port forwarding

### ISSUE THREAD

**Issue by bcnx** [sshuttle/sshuttle#748]
Hi,

my client is only reachable via an outgoing SSH connections. To properly support him, I was thinking to forward the SSH service of an internal SSH server to a public SSH server which I manage. I use this command:

ssh -R 0.0.0.0:6666:localhost:22 ssh.public.server

The next step would be to launch sshuttle like this:

sshuttle -r root@ssh.public.server:6666 192.168.1.0/24

This however does not work. After launching the second command I cannot do a ping (could be normal if only TCP traffic is supported) or an nmap scan (should give some results).

Is this at all possible?

Cheers,

BC

**Comment by rushic24** [comment]
Hi, were you able to resolve this?

**Comment by bcnx** [comment]
No, I believe in the end I configured OpenVPN for TCP and forwared that port.

BC
